components/org.wso2.carbon.identity.data.publisher.authentication.audit/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.utils</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.user.core</artifactId>
+        </dependency>
```

Comment:
why do we need this?

Reply:
getInitatorId(String username, String tenantDomain) was moved into IdentityUtils as suggested. That's why this dependency is needed.